cg/apps/demultiplex/sample_sheet/models.py

```diff
@@ -3,16 +3,16 @@
 from typing import List
 
 from cg.constants.constants import GenomeVersion
-from cg.constants.demultiplexing import SampleSheetHeaderColumnNames
+from cg.constants.demultiplexing import SampleSheetNovaSeq6000Sections
 
 LOG = logging.getLogger(__name__)
 
 
 class FlowCellSample(BaseModel):
     """This model is used when parsing/validating existing sample sheets."""
 
-    flowcell_id: str = Field(..., alias=SampleSheetHeaderColumnNames.FLOW_CELL_ID.value)
-    lane: int = Field(..., alias=SampleSheetHeaderColumnNames.LANE.value)
+    flowcell_id: str = Field(..., alias=SampleSheetNovaSeq6000Sections.Data.FLOW_CELL_ID.value)
+    lane: int = Field(..., alias=SampleSheetNovaSeq6000Sections.Data.LANE.value)
```

Comment:
Is this class specifically for the `NovaSeq6000` sample sheets?

Reply:
yes! In the next PR specific sample models will be derived from this class

Reply:
Ok! So the sample sheet generation differs based on the sequencing platform used? I should already know this 😅

Reply:
Ok! Should the class name `FlowCellSample` reflect that it is for NovaSeq6000?

Reply:
Right now is not necessary because all sample sheet generation is for NovaSeq6000. I will wait for that implementation to rename the class

Reply:
Aha, I understand! Thanks for explaining